src/github.com/getlantern/flashlight/logging/logging.go

```diff
@@ -67,20 +67,23 @@ func Init() error {
 }
 
 func Configure(addr string, cloudConfigCA string, instanceId string,
-	version string, revisionDate string) {
+	version string, revisionDate string) (done chan bool) {
 	if logglyToken == "" {
-		log.Debugf("No logglyToken, not sending error logs to Loggly")
-		return
+		//log.Debugf("No logglyToken, not sending error logs to Loggly")
+		logglyToken = "testLogglyToken"
+		//return
```

Comment:
Ok, sorry for this leftover. I'm using my own "local loggly server" (just an `nc` server) so I had this overwritten for testing. Will revert in final commit.